app/models/user.rb

```diff
@@ -3,6 +3,9 @@ class User < ApplicationRecord
   has_many :comments, foreign_key: 'author_id'
   has_many :posts, foreign_key: 'author_id'
 
+  validates :name, presence: true
+  validates :post_counter, numericality: { only_integer: true }, comparison: { greater_than_or_equal_to: 0 }
+
   def recent_posts
     posts.order(created_at: :desc).limit(3)
   end
```

Comment:
- [ ] Please kindly implement unit test for this method as well according to the project requirement. 👍